addons/pos_restaurant/static/src/js/floors.js

```diff
@@ -292,6 +292,13 @@ var TableWidget = PosBaseWidget.extend({
             }
             floorplan.update_toolbar();
             self.destroy();
+        }, function(err, event) {
+            self.gui.show_popup('error', {
+                'title':_t('Changes could not be saved'),
+                'body': _t('Check your internet connection and access rights'),
```

Comment:
In this case, the user shouldn't even be able to edit the table plan. The "pen" icon shouldn't be visible.